docs/teleport/aks-getting-started.md

```diff
@@ -1,78 +1,59 @@
-# Getting Started with AKS & Project Teleport
+---
+title: Integrate Azure Container Registry and Project Teleport with Azure Kubernetes Service
+description: Learn how to integrate Azure Kubernetes Service (AKS) with Azure Container Registry (ACR) and Project Teleport
+services: container-service
+ms.topic: article
+ms.date: 02/22/2021
+---
 
-> Note: this is a first draft, to get folks started.
+# Integrate Azure Container Registry and Project Teleport with Azure Kubernetes Service (preview)
 
-To get a sense of the performance benefits of Project Teleport two deployments will be made.
+[Project Teleport][project-teleport] allows container hosts to access pre-expanded layers within an [Azure Container Registry (ACR)][acr] that is in the same region as the container host. Using pre-expanded layers removes the time for compute and memory to decompress layers that are already available within the Azure network. Removing this decompression also reduces the time to create the instance of the running container.
```

Comment:
add references for named links at the bottom. You need to do this throughout the doc